edx_rest_framework_extensions/auth/jwt/tests/test_authentication.py

```diff
@@ -179,45 +178,21 @@ def test_authenticate_credentials_no_usernames(self):
     _MOCK_ANONYMOUS_RETURN = None
     _MOCK_USER_AUTH_RETURN = ('mock-user', "mock-auth")
 
-    @ddt.data(
-        # CSRF exempt because roll-out is not enabled
-        (False, False, _MOCK_USER_AUTH_RETURN),
-        # CSRF exempt because roll-out is not enabled (even though JWT cookies are used)
-        (False, True, _MOCK_USER_AUTH_RETURN),
-        # CSRF exempt because of anonymous access (similar to SessionAuthentication)
-        (True, True, _MOCK_ANONYMOUS_RETURN),
-        # CSRF exempt because request uses JWT authentication without JWT cookies
-        (True, False, _MOCK_USER_AUTH_RETURN),
-    )
-    @ddt.unpack
-    def test_authenticate_csrf_exempt(self, enable_rollout, use_jwt_cookies, mocked_return_value_user_and_auth):
-        """ Verify authenticate success for cases that are CSRF exempt. """
-        request = RequestFactory().post('/')
-        if use_jwt_cookies:
-            request.META[USE_JWT_COOKIE_HEADER] = 'true'
-
-        with mock.patch.object(JSONWebTokenAuthentication, 'authenticate', return_value=mocked_return_value_user_and_auth):  # noqa E501 line too long
-            with override_settings(EDX_DRF_EXTENSIONS={ENABLE_ANONYMOUS_ACCESS_ROLLOUT: enable_rollout}):
-                actual_user_and_auth = JwtAuthentication().authenticate(request)
-
-        self.assertEqual(mocked_return_value_user_and_auth, actual_user_and_auth)
-
     @ddt.data(
         # CSRF protected because using JWT cookies to successfully authenticate (similar to SessionAuthentication)
-        (True, True, _MOCK_USER_AUTH_RETURN),
+        (True, _MOCK_USER_AUTH_RETURN),
```

Comment:
Do we need DDT if there is only one set of params here?  Can we just flatten this out?